types/zeromq/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for zeromq 4.6
+// Type definitions for zeromq 5.2
 // Project: https://github.com/zeromq/zeromq.js
 // Definitions by: Dave McKeown <https://github.com/davemckeown>
 //                 Erik Mavrinac <https://github.com/erikma>
```

Comment:
@overflowz Do you want to add your name here?

Reply:
done 👌